opendebates/tests/test_navigation.py

```diff
@@ -0,0 +1,53 @@
+from django.core.urlresolvers import reverse
+from django.test import TestCase
+from django.test.utils import override_settings
+
+import re
+
+from .factories import SubmissionFactory
+
+
+@override_settings(SUBMISSIONS_PER_PAGE=2)
+class NavigationTest(TestCase):
+
+    def setUp(self):
+        self.url = reverse('list_ideas')
+        self.search_url = reverse('search_ideas')
+
+        SubmissionFactory(votes=1)
+        SubmissionFactory(votes=2)
+        SubmissionFactory(headline="Something about ducks", votes=10)
+        SubmissionFactory(headline="Another thing about ducks", votes=5)
+
+    def find_first_page_link(self, content):
```

Comment:
Whoops - it's a helper method I inadvertently copied and pasted from another test file.  No need for it here, I'll remove.